scripts/teamDataScript.js

```diff
@@ -232,55 +232,71 @@ function savePitAnswers() //onclick for save pit button
     
     $('#pitTableConfirmation').text('Answers saved.');
     showAllPitDataTable();
-}
+}   
 
 function showAllPitDataTable() //shows pit stats of all teams for the current event in one table
 {
     var season = getCurrSeason();
     var event = getCurrEvent();
     var cname = season + ' pitQuestions';
-    var questionsArr = getCookie(cname).split('Ω').slice(0,-1); //first element to second to last element of .split arr
-    $('#allTeamsPitAnswers').html('');
+    var questionsArr = getCookie(cname).split(COOKIE_QUESTION_SEPARATOR).slice(0,-1); //first element to second to last element of .split arr
+    
+    var initialTableHTML = '<tr class="teamRow"><th class="allPitTableHeader">Team #</th></tr>';
+    $('#allTeamsPitAnswers').html(initialTableHTML);
     $('#allPitAnswersTitle').text(`All Pit Scouting Data for the ${event} event.`);
 
-    var tableHeader = '<tr><th class="allTeamsRow">Team #</th>'; //top-left most cell
-    for (var i = 0; i < questionsArr.length; i++)
+    //adding initial rows with question headers
+    for (var i = 0; i < questionsArr.length; i++) //for each question
     {
-
-        tableHeader += `<th>${questionsArr[i]}</th>`;
+        var questionWithNoSpaces = stripAlphaNumerics(questionsArr[i]);
+        var questionHeaderHTML = `<th class="allPitTableHeader">${questionsArr[i]}</th>`;
+        var newRowHTML = `<tr class='${questionWithNoSpaces}'>${questionHeaderHTML}</tr>`; //the question is one of the classes (w/o whitespaces), will help with adding team data
```

Comment:
this is probably a case where we're streaching the class use a little too far.

If you're setting this to easily find the right row to edit, then make it's own attribute.

```js
`<tr question='${questionWithNoSpaces}'>${questionHeaderHTML}</tr>`
```
then to find this row, use 
```js
$(`tr[question='${questionWithNoSpaces}']`)
```
to find it. (this has the benefit of no you can use the question itself, without modifying it to string spaces and such out).